calicoctl/commands/delete.go

```diff
@@ -25,6 +25,9 @@ import (
 func Delete(args []string) error {
 	doc := EtcdIntro + `Delete a resource identified by file, stdin or resource type and name.
 
+Valid resource kinds are bgpPeer, hostEndpoint, policy, pool, profile and tier.  The <KIND>
```

Comment:
The reason I added them as camel-case here was because the YAML/JSON is case sensitive - so this makes it less confusing if you are using those.